src/c/BuscaSequencial.c

```diff
@@ -0,0 +1,19 @@
+#include <stdio.h>
+
+int buscaSequencial(int vetor[], int buscado) {
+    size_t n = sizeof(vetor[0]);
```

Comment:
Para pegar o tamanho do array tem que pegar o tamanho em bytes e dividir pelo tamanho do tipo (e.g. int):

```suggestion
    size_t n = sizeof(vetor) / sizeof(vetor[0]);
```

Reply:
Obrigada, @kelvins! Clareou diversas dúvidas hehe.